.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first PR, read our contributor guidelines https://jenkins-x.io/contribute/
+2. Follow these instructions to write commit messages http://karma-runner.github.io/3.0/dev/git-commit-msg.html
+3. Follow these instructions to write tests https://jenkins-x.io/contribute/development/#testing
+4. You can trigger the tests for your PR with /test bdd
+5. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**What this PR does / Why we need it**:
+
+**Does your PR have tests**:
```

Comment:
suggest rewording to a checkbox?

`[ ] Change is covered by existing or new tests`